pkg/apic/provisioning/propertybuilder.go

```diff
@@ -464,43 +491,43 @@ func (p *arraySchemaProperty) Build() (def *propertyDefinition, err error) {
   object property datatype builder
 */
 // objectSchemaProperty - adds specific info needed for an object schema property
-// type objectSchemaProperty struct {
```

Comment:
Can this code be removed?

Reply:
I just uncommented it, streams will be using it